cmd/serve.go

```diff
@@ -80,23 +78,31 @@ func (s *ServerService) GetSchedulerJobRegistrars(ctx context.Context, serveOpts
 
 	return []scheduler.SchedulerJobRegisterOption{
 		scheduler.WithPaymentToSubmitterJobOption(models),
-		scheduler.WithPaymentFromSubmitterJobOption(models),
 		scheduler.WithAPAuthEnforcementJob(apAPIService, serveOpts.MonitorService, serveOpts.CrashTrackerClient.Clone()),
 		scheduler.WithPatchAnchorPlatformTransactionsCompletionJobOption(apAPIService, models),
 		scheduler.WithReadyPaymentsCancellationJobOption(models),
 	}, nil
 }
 
-func (s *ServerService) SetupConsumers(ctx context.Context, serveOpts serve.ServeOptions, eventHandlerOptions events.EventHandlerOptions) {
+func (s *ServerService) SetupConsumers(ctx context.Context, eventBrokerOptions utils.EventBrokerOptions, eventHandlerOptions events.EventHandlerOptions, serveOpts serve.ServeOptions) TearDownFunc {
```

Comment:
I liked the approach of closing connections in case of error (good catch). Also, I'll return an error instead of the current rather than calling `log.Fatal`.